src/masternodes/govvariables/attributes.cpp

```diff
@@ -1049,6 +1174,29 @@ Res ATTRIBUTES::Validate(const CCustomCSView & view) const
                 }
             break;
 
+            case AttributeTypes::Consortium:
+                switch (attrV0->key) {
+                    case ConsortiumKeys::Members:
```

Comment:
ConsortiumKeys::Members and ConsortiumKeys::MintLimit have identical validation, can remove the code and the break under ConsortiumKeys::Members to apply the same code for both.

```
case ConsortiumKeys::Members:
case ConsortiumKeys::MintLimit:
```